validator/client/validator_propose.go

```diff
@@ -51,6 +52,19 @@ func (v *validator) ProposeBlock(ctx context.Context, slot uint64, pubKey [48]by
 		return
 	}
 
+	history, err := v.db.ProposalHistory(pubKey[:])
+	if err != nil {
+		log.WithError(err).Error("Failed to get proposal history")
+		return
+	}
+
+	if db.HasProposedForEpoch(history, epoch) {
+		log.Info("Tried to sign a double proposal, rejected")
```

Comment:
Used an info log here since it isn't technically an error, is that fine?

Reply:
Change to WARN and add specific data to the log (what epoch am I attempting to double sign)

Reply:
+1